packages/p2p/src/utils/advertiser.ts

```diff
@@ -0,0 +1,31 @@
+import { localize } from 'Components/i18next';
+import { isMobile } from '@deriv/shared';
+
+/**
+ * Function to get the message to be displayed in the blocked users modal
+ * @function getMessage
+ * @param {number} user_blocked_count // the numbers of users who have blocked the current user
+ * @return { string }
+ * */
+
+export const getMessage = (user_blocked_count: number): string => {
+    switch (user_blocked_count) {
+        case 0:
+            return localize('Nobody has blocked you. Yay!');
+        case 1:
+            return localize('{{user_blocked_count}} person has blocked you', {
+                user_blocked_count,
+            });
+        default:
+            return localize('{{user_blocked_count}} people have blocked you', {
+                user_blocked_count,
+            });
+    }
+};
+
+/**
+ * Function to get the label size based on the screen size
+ * @function getLabelSize
+ * @returns {string} label size
+ */
+export const getLabelSize = (): string => (isMobile() ? 'xxxs' : 'xs');
```

Comment:
Perhaps we can make it able to accept different sizes so we can in the future use it in other places, since a lot of pages and components use this type of logic. I've done something similar in the my profile refactor card, where i created a responsive.ts utils file with the `getTextSize` function :) 

![Screenshot 2023-07-18 at 10 22 18 AM](https://github.com/farrah-deriv/deriv-app/assets/103412909/2d804ea7-428c-4bba-9b2d-a9dc0ed24c9b)